migrations/2023-07-07-091223_create_captures_table/up.sql

```diff
@@ -0,0 +1,25 @@
+
+CREATE TYPE "CaptureStatus" AS ENUM (
+    'started',
+    'charged',
+    'pending',
+    'failure'
+);
+CREATE TABLE captures(
+    id SERIAL PRIMARY KEY,
```

Comment:
Check if we want to autogenerate id's. We don't do this as its problematic during migration of data.